nox/command.py

```diff
@@ -124,6 +124,9 @@ def run(
 
             raise CommandFailed("Returned code {}".format(return_code))
 
+        if output:
+            logger.debug(output)
```

Comment:
So the problem with using `logger.debug` is that the output will actually differ from the normal operation because the logger appends `nox >` to the begging of each log-line. I would prefer that this goes to `stderr` just like the block above that sais `if silent: sys.stderr.write(output)`.